src/audio/pipeline.c

```diff
@@ -247,8 +247,6 @@ struct pipeline *pipeline_new(struct sof_ipc_pipe_new *pipe_desc,
 	schedule_task_init(&p->pipe_task, pipeline_task, p);
 	schedule_task_config(&p->pipe_task, pipe_desc->priority,
 		pipe_desc->core);
-	list_init(&p->comp_list);
```

Comment:
Can you add some more detail in the commit message stating what APIs are duplicated and presumably that this patch merges these into a single API ?

Reply:
Will do.